doc.py

```diff
@@ -240,7 +240,7 @@ def write_action_command_usage(self, documentation_generator: ToolDocumentationG
         self._write_command_usage(CommandUsageGenerator(documentation_generator).get_action_command_usage(action))
 
     def _write_command_usage(self, lines):
-        self.file.new_header(level=3, title='Usage')
+        self.file.new_header(level=3, title='Usage', add_table_of_contents='n')
```

Comment:
needed because otherwise it breaks (no header with previous level (2) to generate content table)
https://github.com/didix21/mdutils/issues/48#issuecomment-738326106